libs/domain/search/src/services/revealers/products-experience-data.revealer.ts

```diff
@@ -0,0 +1,34 @@
+import { inject } from '@spryker-oryx/di';
+import {
+  catchMessage,
+  ExperienceDataRevealer,
+  MessageType,
+  postMessage,
+} from '@spryker-oryx/experience';
+import { Observable, of, switchMap, tap } from 'rxjs';
+import { Suggestion } from '../../models';
+
+import { SuggestionService } from '../suggestion';
+
+export class ProductsExperienceDataRevealer implements ExperienceDataRevealer {
```

Comment:
I've tried to use the product sku editor inside EB and it no longer works. not sure if it's related to this PR though.

Reply:
It's not releated to this PR. we have bug in EB